### ISSUE TITLE
Content-owner - bug on search/profile not displayed

### ISSUE THREAD

**Issue by Kimberley-Heath** [emfoundation/ce100-app#1332]
Content Owners do not come up on search for Primary/Secondary users, only admin. They are however visible on the Organisations people list. 

![screen shot 2018-05-31 at 10 54 06](https://user-images.githubusercontent.com/36198201/40776014-737e2e3e-64c1-11e8-91c8-055ad2be2b06.png)

![screen shot 2018-05-31 at 10 54 29](https://user-images.githubusercontent.com/36198201/40776024-7d12d8be-64c1-11e8-8d0e-6c4320a5d2d1.png)

As an admin or Primary user, neither can visit content-owners profiles: they get a 404 issue.

![screen shot 2018-05-31 at 10 54 59](https://user-images.githubusercontent.com/36198201/40775933-32086852-64c1-11e8-968a-388c940f4401.png)

@asimonw to review content-owners as a user type before fixing this issue...

**Comment by asimonw** [comment]
@Kimberley-Heath Agreed that content owners and admins shouldn't show up on org profiles.

We can fix this to ensure that this doesn't accidentally happens, however admins and content owners (and view only roles) shouldn't be attached to organisations. They are supposed to be "backend" roles.
A short term mitigation is to remove that content owner from the org.

This confusion is understandable however, and we're thinking about decoupling frontend and backend roles, so that one person can have both a frontend and backend role (useful for EMFers). Only the frontend role would be linked to an org, challenges etc, and have a public profile.

**Comment by asimonw** [comment]
@Kimberley-Heath and @fannyBkane based on what I said above, I'll create a new issue to cover the bigger question, namely how to treat backend roles going forward.
